File: redsheet-portal/src/app/database-models/meeting-requirement.ts
import { required, maxLength, range } from '@rx/annotations';

export class MeetingRequirement {
    constructor(meetingRequirement?: MeetingRequirement )  {
        let properties = [ "cultureCountryId", "meetingRequirementId", "meetingRequirementValue",];
        for (let property of properties)
            if (meetingRequirement && meetingRequirement[property])
                this[property] = meetingRequirement[property];
    }
 
    @range(1,2147483647)
	cultureCountryId : number =   undefined;
 
	meetingRequirementId : number =   0 ;
 
    @required()
    @maxLength(500)
	meetingRequirementValue : string =   undefined;
}

File: RedSheet.DbEntities/Models/MeetingRequirement.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("MeetingRequirements", Schema = "dbo")]
	[Serializable]
	public partial class MeetingRequirement  
	{
 
	[Range(1, int.MaxValue)]
	public int CultureCountryId { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int MeetingRequirementId { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string MeetingRequirementValue { get; set; } 

	public MeetingRequirement()
	{
    }

    }
}

